pages/api/members.ts

```diff
@@ -39,14 +65,35 @@ export default async function handler(req, res) {
     .map((member: NotionMember) => ({
       name: member?.properties?.Name?.title?.at(0)?.plain_text,
       image: member?.icon?.file?.url,
-      roles: member?.properties?.Role.multi_select.map((role) => role.name),
+      roles: member?.properties?.Role.multi_select
+        .map((role) => role.name)
+        .sort((a, b) => {
+          if (a == "President" && b !== "President") {
```

Comment:
I think the best way to create a comparator based on a string order e.g. "president" > "teamlead" is to create a hashmap

```ts
const roleOrder = {
  "president" : 0,
  "teamlead" : 1,
  "member" : 2,
}

const comparator = (a, b) => (roleOrder[a] - roleOrder[b])
```